pkg/extensions/extension_search_test.go

```diff
@@ -31,7 +32,7 @@ func TestTrivyDBGenerator(t *testing.T) {
 		defer os.Remove(logFile.Name()) // clean up
 
 		logger := log.NewLogger("debug", logFile.Name())
```

Comment:
Please use a logger which sends messages to both stdout and the logfile?
````
writers := io.MultiWriter(os.Stdout, logFile)
logger.Output(writers)
```
There are some tests using similar logic in search_test.go.
It's hard to figure out why the test is failing if the logs are not shown in the console.